### DISCUSSION TITLE
JavaScript-based Gomplate Alternative

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [hairyhenderson/gomplate#1733] [Q&A] [answered]
Hey, I love Gomplate but I feel like a JavaScript-based templating system would better suite me. Does anyone know of any JavaScript based templating tools?

**Comment by hairyhenderson** [comment]
I think this question would be best asked elsewhere 😉

**Comment by ProfessorManhattan** [comment]
Hey @hairyhenderson -- Why? Many of the best open source projects list pros / cons of alternatives somewhere in their documentation and gomplate is definitely if not the best template lib

**Reply by hairyhenderson** [reply, marked as answer]
If other projects take the time to compare themselves and document these comparisons, that's their choice. However, for gomplate I neither have the time not the interest in spending time doing competitive analysis against other open-source projects.

If it benefits you, great. If another tool is more appropriate, that's totally fine.
